baselines/powerbi.md

```diff
@@ -137,9 +83,9 @@ security groups, instead of allowing the entire agency to publish data
 to the web.
 
 ### Policies
-#### MS.POWERBI.2.1v1
-The Publish to Web feature SHOULD be disabled unless the agency mission requires the capability.
-- _Rationale:_ TODO
+#### MS.POWERBI.1.1v1
+Disable the Publish to Web feature unless the agency mission requires the capability.
```

Comment:
"shall" and "should" language is still expected in the policies. Only removed from the previous "policy groups".

Reply:
left this comment only for the first policy to not create duplicate comments